### ISSUE TITLE
Base on native ResizeObserver

### ISSUE THREAD

**Issue by kelunik** [Akryum/vue-resize#205]
In the future, this package could be directly based on https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver.

**Comment by Akryum** [comment]
Only when we will not want to support older browsers anymore :D

**Comment by kelunik** [comment]
Yes, of course. Can this be based on targets and different builds?

**Comment by matxx** [comment]
I was looking for this
I found an alternative using `ResizeObserver ` : https://vueuse.org/core/useResizeObserver
